components/SubRow.module.css

```diff
@@ -1,16 +1,17 @@
 .tableCellStyle{
     font-family: var(--secondary-font-family);
-    font-size: var(--font-size-large);
-	background-color: var(--colour-container-background);
-	color: var(--colour-font)
+    font-size: 1.1rem;
+	background-color: #f5f5f5;
+	color: var(--colour-font);
+	border-color: #7a7a7a;
```

Comment:
again I think maybe add these to the global css file and try to standardise - maybe there is a reason you have it this way though?